nat-lab/tests/test_dns.py

```diff
@@ -190,36 +190,58 @@ async def test_dns_port(alpha_ip_stack: IPStack) -> None:
         connection_alpha, _ = [conn.connection for conn in env.connections]
 
         # These call should timeout without returning anything
-        with pytest.raises(asyncio.TimeoutError):
-            await testing.wait_normal(
+        with pytest.raises(ProcessExecError):
+            await testing.wait_long(
                 connection_alpha.create_process(
-                    ["dig", "@" + dns_server_address_alpha, "-p", "53", "google.com"]
+                    [
+                        "dig",
+                        "+timeout=1",
+                        "@" + dns_server_address_alpha,
+                        "-p",
+                        "53",
+                        "google.com",
+                    ]
                 ).execute()
             )
```

Comment:
What do you think about extracting this to its own function, either in this file or `dns.py`?

Reply:
I think is not that benefitial like with `nslookup` on `query_dns`, it wouldn't hurt though..  will do it

Reply:
For sure not as beneficial since it's only used in one test, but if we need to change how we're calling `dig`, it's better to just have to change in one place